docs/cookbook/support_ticket_routing.md

```diff
@@ -0,0 +1,232 @@
+# Support Ticket Routing
+
+This recipe shows how to take an incoming support ticket/call transcript then use an LLM to summarize the issue and route it to the correct person.
+
+??? info "Key Concepts"
+
+    - [Prompts](../learn/prompts.md)
+    - [Calls](../learn/calls.md)
+    - [Tools](../learn/tools.md)
+    - [Dynamic Configuration](../learn/dynamic_configuration.md)
+    - [Chaining](../learn/chaining.md)
+    - [Response Models](../learn/response_models.md)
+
+!!! note "Background"
+    Traditional machine learning techniques like text classification were previously used to solve routing. LLMs have enhanced routing by being able to better interpret nuances of inquiries as well as using client history and knowledge of the product to make more informed decisions.
+
+## Imitating a Company’s Database/Functionality
+
+!!! Note
```

Comment:
nit: should be "note", also what about a title? or no title?

Reply:
no title i think?

Reply:
ok, still update then to `!!! note ""`